packages/server/test/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../../tsconfig.packages.json",
+  "compilerOptions": {
+    "rootDir": ".."
+  },
+  "include": ["../test/**/*", "../src/**/*"]
+}
```

Comment:
@laurentlp did you remove this config? It's necessary to have intelisense with vscode

Reply:
Right, I'll make sure to keep it!